### PR TITLE
Lock minio version

### PR DESCRIPTION
**Pull request by arcusfelis**
This PR addresses "unstable CI".

Proposed changes include:
* Lock version of Minio and Minio Client
* Don't use link flag (podman does not support that legacy way and docker discourages of using it too)

### CODE REVIEW

**Comment by mongoose-im** [comment]
[small_tests_23](https://circleci.com/gh/esl/MongooseIM/104495) / small_tests / 2ab817d2c144f71f4d289eef89a3388d04d31f79
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3530/104495/small_tests.23.1) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3530/104495/small_tests.23.1/small/ct_run.mongooseim@localhost.2022-02-02_12.12.17/lib.mongooseim.logs/run.2022-02-02_12.12.17/suite.log.html)

**Review by chrzaszcz — APPROVED** [review]
Looks good

File: tools/db-versions.sh [hunk 1]
```diff
@@ -1,15 +1,35 @@
-# Default cassandra version
-CASSANDRA_VERSION=${CASSANDRA_VERSION:-3.9}
+# Versions
```
Comment (NelsonVides) [inline tools/db-versions.sh:1]: Doesn't this way of defining versions and then overriding them feel a bit redundant? Why's that better than the previous `VERSION=${VERSION:-XX.YY}`?

**Review by NelsonVides — COMMENTED** [review]
Perfect, just a stylistic question

File: tools/db-versions.sh (hunk 1 above)
Comment (arcusfelis) [reply tools/db-versions.sh:1]: It makes it easier to read (i.e. once version contains `-`, it kinda becomes a mess).

Comment (chrzaszcz) [reply tools/db-versions.sh:1]: I agree it's verbose. I merged the PR because we need tests fixed asap.

**Pull request merged**